src/solver/opamCudf.ml

```diff
@@ -1144,20 +1144,13 @@ let extract_explanations packages cudfnv2opam reasons : explanation list =
     reduce_invariant_msg [] false explanations
   in
 
-  let same_depexts sdeps fdeps =
-    List.for_all (function
-        | `Missing (_, sdeps', fdeps') -> sdeps = sdeps' && fdeps = fdeps'
-        | _ -> false)
-  in
   log ~level:3 "Explanations: %a" Pp_explanation.pp_explanationlist explanations;
   match explanations with
   | [] ->
     OpamConsole.error_and_exit `Internal_error
       "Internal error while computing conflict explanations:\n\
        sorry about that. Please report how you got here in \
        https://github.com/ocaml/opam/discussions/5130 if possible."
-  | `Missing (_, sdeps, fdeps) :: rest when same_depexts sdeps fdeps rest ->
```

Comment:
Is it possible to display differently the dependency path/conflict? Like that the focus remain at first look on the missing package, and the information of the dependency is available.

Reply:
i don't understand that

Reply:
before we had:
```
  * Missing dependency:
    - a
```
Now we will have
```
  * Missing dependency:
    - c -> b -> a
```
I was proposing to display differently `a` from the prefixed path `c -> b ->`. Like that at first look it is directly visible that the main information is `a`.
It is kind of UI trick, i didn't see on a terminal how it is, I though about it while looking at the test, which are not colored.

Reply:
In that case i disagree. I think the reason for the failure should match the actual reason. Adding special cases like this would complicate the code and confuse users.

Reply:
I don't want to hide, but to highlight. For example is `a` (the main missing) is bold, we would have
![image](https://github.com/user-attachments/assets/bade313a-979d-43a7-a20c-488093b20b64)
We have all the information (the full path), with a highlight on the main reason (missing package).

Reply:
I'm not sure i follow. The highlight is already done, the last item on the list is already bold and red when displayed in an interactive terminal